configure.ml

```diff
@@ -109,7 +109,7 @@ let configure bindir sbindir libexecdir scriptsdir etcdir mandir =
   let configmllines =
     [ "(* Warning - this file is autogenerated by the configure script *)";
       "(* Do not edit *)";
-      Printf.sprintf "#let xen45 = %b" xen45
+      Printf.sprintf "#define xen45 %d" (if xen45 then 1 else 0) (* cppo is a bit broken *)
```

Comment:
So `#let` is no longer supported by cppo? I think the solution is fine. Another option would have been to define or un-define`xen45` and use `#ifdef` in other parts of the code.